pyface/action/schema/tests/test_action_manager_builder.py

```diff
@@ -0,0 +1,443 @@
+# (C) Copyright 2005-2022 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from contextlib import contextmanager
+import unittest
+
+
+from pyface.action.api import (
+    Action,
+    ActionItem,
+    ActionManager,
+    Group,
+    MenuManager,
+    MenuBarManager,
+)
+from pyface.tasks.action.api import (
+    GroupSchema,
+    MenuSchema,
+    MenuBarSchema,
+    SchemaAddition,
+)
```

Comment:
```suggestion
from pyface.action.schema.api import (
    GroupSchema,
    MenuSchema,
    MenuBarSchema,
    SchemaAddition,
)
```

We probably don't want a `pyface.tasks` import here